File: app/src/main/java/com/meteorhead/leave/leavelist/LeaveViewHolder.java
package com.meteorhead.leave.leavelist;

import android.support.v7.widget.RecyclerView;

import com.meteorhead.leave.databinding.LeaveListItemBinding;

/**
 * Created by Lenovo on 2016-09-13.
 */

public class LeaveViewHolder extends RecyclerView.ViewHolder {
    private LeaveListItemBinding itemBinding;

    public LeaveViewHolder(LeaveListItemBinding viewBinding) {
        super(viewBinding.getRoot());
        itemBinding = viewBinding;
    }

    public LeaveListItemBinding getItemBinding() {
        return itemBinding;
    }
}

File: app/src/main/java/com/meteorhead/leave/screens/bestleaves/list/LeaveViewHolder.kt
package com.meteorhead.leave.screens.bestleaves.list

import android.support.v7.widget.RecyclerView
import com.meteorhead.leave.data.Leave
import com.meteorhead.leave.databinding.BestLeaveItemBinding

/**
 * Created by wierzchanowskig on 01.03.2017.
 */
class LeaveViewHolder(var binding: BestLeaveItemBinding): RecyclerView.ViewHolder(binding.root) {
    fun bind(leave: Leave) {
        binding.viewModel.leave = leave
    }
}